source/tests/system/nidcpower_driver_api_tests.cpp

```diff
@@ -70,6 +70,19 @@ class NiDCPowerDriverApiTest : public ::testing::Test {
     ASSERT_EQ(kdcpowerDriverApiSuccess, response.status());
   }
 
+  void initiate()
```

Comment:
note: This helper is being added as part of other pull request. Will remove it from here once other changes gets merged with `features\ni-dcpower` branch.